pymc/step_methods/arraystep.py

```diff
@@ -41,11 +39,12 @@ class ArrayStep(BlockedStep):
     blocked: Boolean (default True)
     """
 
-    def __init__(self, vars, fs, allvars=False, blocked=True):
+    def __init__(self, vars, fs, allvars=False, blocked=True, rng: RandomGenerator = None):
```

Comment:
```suggestion
    def __init__(self, vars, fs, allvars=False, blocked=True, rng: RandomGenerator | None = None):
```
Also in other places

Reply:
The `RandomGenerator` type already includes `None`